src/components/Contentful/TestimonialCards.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+	<div>
+		<section-with-background :background-content="background" class="tw-bg-gray-50">
+			<template #content>
+				<kv-page-container>
+					<kv-grid class="tw-grid-cols-12 tw-text-center">
+						<h2
+							v-html="testimonialHeadline"
+							class="tw-py-4 tw-col-span-12"
+						>
+						</h2>
+
+						<div
+							v-for="(singleCard, index) in cardData"
+							:key="index"
+							class="tw-col-span-12 md:tw-col-span-6 lg:tw-col-span-6
+							tw-mb-2
+							tw-bg-white tw-rounded"
+						>
+							<div class="tw-pt-4 tw-pb-3">
+								<kv-contentful-img
+									v-if="singleCard.imageUrl"
+									class="tw-rounded-full tw-overflow-hidden"
+									:class="'testimonial-card-' + index"
+									:contentful-src="singleCard.imageUrl"
+									:alt="singleCard.parsedName"
+									:width="64"
+									:height="64"
+									loading="lazy"
+									fallback-format="jpg"
+								/>
+							</div>
+
+							<p
+								v-html="singleCard.quote"
+								class="tw-text-subhead tw-pb-4 tw-px-2"
+							>
+							</p>
+
+							<h3
+								v-html="singleCard.name + ','"
+								class="tw-pb-2"
+							>
+							</h3>
+
+							<h4
+								v-html="singleCard.title"
+								class="tw-text-gray-500 tw-pb-4"
+							>
+							</h4>
+						</div>
+					</kv-grid>
+				</kv-page-container>
+			</template>
+		</section-with-background>
+	</div>
+</template>
+
+<script>
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import SectionWithBackground from '@/components/Contentful/SectionWithBackground';
+import { richTextRenderer } from '@/util/contentful/richTextRenderer';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+import KvGrid from '~/@kiva/kv-components/vue/KvGrid';
+
+export default {
+	components: {
+		KvContentfulImg,
+		KvGrid,
+		KvPageContainer,
+		SectionWithBackground,
+	},
+	props: {
+		content: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		contentfulComponentData() {
+			const contentfulData = this.content.contents.filter(({ contentType }) => {
+				return contentType === 'genericContentBlock';
+			});
+			return contentfulData;
+		},
+		testimonialHeadline() {
+			return this.contentfulComponentData[0].headline;
+		},
+		cardData() {
+			const allCardData = [];
+			this.contentfulComponentData.forEach(arrayItem => {
```

Comment:
This is very readable and clean, nice work. One small improvement would be to use the "map" array method instead of foreaching and pushing into the new array. Not necessary for this PR but a handy thing to learn since map is prevelant in a lot of  codebases.

Reply:
Thanks for this suggestion @ryan-ludwig, update was made in the lastest code commit.